### PR TITLE
Fix command errors and README.md update

### PR DESCRIPTION
**Pull request by Marcerisson**
- openssl rand was not in the right format
- list of commands in README.md weren't working

### CODE REVIEW

**Comment by seirl** [comment]
Thanks! Can you prefix your commit messages with the name of the affected part of the project? like `README: blahblah` or `settings: blahblah`. I'll merge it after this change.

**Comment by Marcerisson** [comment]
there we go!
spent the last 2 hours trying to figure out how to change a commit's name..

**Comment by seirl** [comment]
Great! Now can you rebase your branch on master, to remove that weird merge commit?

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@
 *.pyc
 
 epiquote/settings/*.py
+
+\.venv/
```
Comment (seirl) [inline .gitignore]: This should go in your `.git/info/exclude`, not the gitignore, as it's something personal. Can you remove this commit? Thanks :-)

**Comment by Marcerisson** [comment]
should be okay now

**Pull request merged**